corehq/apps/case_importer/tasks.py

```diff
@@ -279,3 +283,10 @@ def _submit_caseblocks(domain, case_type, caseblocks):
         'errors': errors.as_dict(),
         'num_chunks': num_chunks,
     }
+
+
+total_bytes = datadog_gauge_task(
+    'case_importer.files.total_bytes',
```

Comment:
No reason to make them short that I know of. The way I think of these is that the dot prefixes are used to group related metrics.